Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3
```

Comment:
Looks like some files from `master` made it into this branch. When pulling from master, try using `git pull --rebase` instead of merging everything into a separate commit.

Reply:
I'll clean this up